core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -95,11 +93,6 @@ private[deploy] class Master(
   // After onStart, webUi will be set
   private var webUi: MasterWebUI = null
 
-  private val masterPublicAddress = {
```

Comment:
`masterPublicAddress` add in [Use external addresses in standalone WebUI on EC2.](https://github.com/apache/spark/commit/cdaa0fad51c7ad6c2a56f6c14faedd08fe341b2e) and never used after [SPARK-33774](https://issues.apache.org/jira/browse/SPARK-33774)